source/plugin/scheduler.rst

```diff
@@ -93,6 +94,35 @@ To cancel a task, simply call the ``Task#cancel`` method:
 
 	task.cancel();
 
+If you need to cancel the task, from within the runnable itself, you can instead opt to use a `Consumer<task>` in order to access the task from within.
```

Comment:
Maybe add a brief description of the below example: "The below example will schedule a task that will count down from 60 and cancel itself upon reaching 0" or something similar